README.md

```diff
@@ -48,7 +48,7 @@ more. See [the wiki](https://github.com/neovim/neovim/wiki/Installing-Neovim)!
 
 ### Community
```

Comment:
Twitter and the mailing list are already mentioned at the Community page, so maybe those should be removed too?